src/web/app/components/question-response-panel/question-response-panel.component.ts

```diff
@@ -104,13 +105,11 @@ export class QuestionResponsePanelComponent {
           question.responsesToSelf = responses.responsesToSelf;
           question.hasResponseButNotVisibleForPreview = responses.hasResponseButNotVisibleForPreview;
           question.hasCommentNotVisibleForPreview = responses.hasCommentNotVisibleForPreview;
-        } else {
-          question.hasResponse = false;
-          if (question.errorMessage) {
-            this.statusMessageService.showSuccessToast('Question '
-              .concat(question.feedbackQuestion.questionNumber.toString())
-              .concat(' has no responses.'));
-          }
```

Comment:
I think it will be best to keep this `else` statement for defensive coding practices. 

The original logic with the `else` ensures that if `responses` is truthy, `if (question.errorMessage) { ... }` will definitely not be evaluated and run. If we remove the `else`, it is possible to display the error message, even if `responses` is truthy, though it probably will not happen. However, let's avoid this situation completely by keeping the original logic.

Feel free to let me know what you think about this!

Reply:
That makes sense to me, thank you for the feedback.